sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/AsynchronousMetricStorage.java

```diff
@@ -107,12 +113,10 @@ void record(Measurement measurement) {
                 start, measurement.epochNanos(), measurement.doubleValue(), processedAttributes)
             : Measurement.longMeasurement(
                 start, measurement.epochNanos(), measurement.longValue(), processedAttributes);
-    recordPoint(aggregator.toPoint(measurement));
+    recordPoint(processedAttributes, measurement); // aggregator.toPoint(measurement));
```

Comment:
Comment to be removed?
```suggestion
    recordPoint(processedAttributes, measurement);
```